src/java/htsjdk/samtools/CRAMIndexer.java

```diff
@@ -187,7 +262,7 @@ private int computeIndexingBin(final Slice slice) {
         /**
          * Record any index information for a given BAM record
          *
-         * @param slice The BAM record. Requires rec.getFileSource() is non-null.
+         * @param slice CRAM slice, single ref or unmapped only.
          */
         public void processAlignment(final Slice slice) {

```

Comment:
The javadoc param has been changed here, but its still confusing that this is called processAlignment when it takes a slice.